### DISCUSSION TITLE
Money Reward for Custom Challenge completion

### DISCUSSION THREAD

**Discussion by olegcherev** [devactivity-app/devActivity-Community#26] [Ideas & Feature Requests]
Integrate with some payment service to reward the Contributor for completed issue and linked PR.
Need kind of approval from the admin role.

**Comment by damonexsppfx-alt** [comment]
Okay

**Reply by olegcherev** [reply]
Thanks for the note, @damonexsppfx-alt.

This idea is **paid bounties on custom challenges** (issue + linked PR), with an **admin approval** step before money is sent. We have not shipped a payments integration for that yet — challenges, XP, and recognition are live; cash payouts are not.

If you have a use case (OSS bounties, intern bonuses, hackathon prizes), share it here. That helps us decide whether to build it vs. keep using XP/achievements plus an external payout tool.
